bedrock/firefox/templates/firefox/testflight.html

```diff
@@ -1,131 +1,135 @@
+{{ css_bundle('firefox_ios_testflight') }}
 {% endblock %}
 
-{% block site_header_nav %}{% endblock %}
-{% block site_header_logo %}{% endblock %}
-
 {% block content %}
-<section id="intro">
-  <div class="container">
-    {# No more new logos! #}
-    {{ high_res_img('protocol/img/logos/firefox/browser/logo-lg.png', {'alt': 'Firefox for iOS', 'width': '142', 'height': '142', 'id': 'header-logo' }) }}
-
-    <header id="intro-header">
-      {# L10n: line break below for visual formatting only #}
-      <h1>{{ _('Help shape the future <br>of Firefox for iOS') }}</h1>
-
-      <h2>{{ _('Sign up to test pre-release versions to make Firefox even better.') }}</h2>
-    </header>
-
-    <div id="footer-email-errors" class="hidden newsletter-form-extra">
-      <ul class="errorlist"></ul>
-    </div>
-
-    <form id="newsletter-form" class="newsletter-form footer-newsletter-form" action="{{ url('newsletter.subscribe') }}" method="post" data-spinner-color="#fff">
+<main role="main">
+  {% call hero(
+  title=_('Help shape the future <br>of Firefox for iOS'),
+  desc=_('Sign up to test pre-release versions to make Firefox even better.'),
+  class='mzp-t-firefox mzp-t-dark mzp-t-product-firefox',
+  include_cta=False,
+  heading_level=1) %}
+  {% endcall %}
+
+  <aside class="mzp-l-content mzp-t-narrow">
+    <form id="newsletter-form" class="mzp-c-newsletter-form" action="{{ url('newsletter.subscribe') }}" method="post" data-spinner-color="#fff">
       {{ newsletter_form.newsletters|safe }}
       {# test flight program is only available in english #}
       <input type="hidden" name="lang" value="en">
       <input type="hidden" name="source_url" value="{{ request.build_absolute_uri() }}">
 
-      <div id="form-fields">
-        <div id="email-submit-wrapper">
-          <div id="email-wrapper" class="field">
-            <label for="id_email">{{ _('Email') }}</label>
-            {{ newsletter_form.email|safe }}
-          </div>
-          <button class="button button-dark" id="footer_email_submit" type="submit">Submit</button>
-        </div>{#-- /#email-submit-wrapper --#}
-
-        <div id="terms-format-wrapper">
-          <div id="format-wrapper">
-            {{ newsletter_form.fmt|safe }}
-          </div>
-
-          <label class="terms-check-label" for="id_terms">
-            <input id="id_terms" name="terms" required="required" type="checkbox">
-            <span class="title">
-            {% trans url='#terms-conditions' %}
+      <fieldset class="mzp-c-newsletter-content">
+        <div class="mzp-c-form-errors" id="newsletter-errors"></div>
+
+        <p>
+          <label for="id_email">{{ _('Email') }}</label>
+          <input type="email" class="mzp-js-email-field" id="id_email" name="email" required aria-required="true">
+        </p>
+
+        <div id="newsletter-details" class="mzp-c-newsletter-details">
+          <input type="hidden" name="lang" id="id_lang"  value="en" aria-required="false">
```

Comment:
It looks like there's already a duplicate `name="lang"` input above on line 37, so perhaps all you need is to add the correct `id` there?